python/lsst/ts/simactuators/path/stop.py

```diff
@@ -0,0 +1,73 @@
+# This file is part of ts_simactuators.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__all__ = ["stop"]
+
+import math
+
+from . import path
+
+
+def stop(pos0, vel0, t0, max_accel):
+    """Compute a path to stop as quickly as possible.
+
+    Parameters
+    ----------
+    pos0 : `float`
+        Starting position (deg)
+    vel0 : `float`
+        Starting velocity (deg/sec)
+    t0 : `float`
+        Start time of path.
+    max_accel : `float`
+        Maximum allowed acceleration (deg/sec^2)
+
+    Returns
+    -------
+    path : `Path`
+        A path with 1-2 segments, ending with a segment of zero acceleration
+        and zero velocity.
+
+    Raises
+    ------
+    ValueError if |max_accel| <= 0
+    """
+    if max_accel <= 0.0:
+        raise ValueError(f"max_accel={max_accel} < 0")
+
+    if vel0 == 0:
+        return path.Path(path.TPVAJ(t0=t0, pos0=pos0), kind=path.Kind.Stopped)
+
+    tpvajs = []
+    # t0 is typically large enough that small time changes
```

Comment:
What does "typically" mean here? Is this an assumption that should be tested (possibly with an assert)? And if this really is the case, should we be using a more accurate time system like `astropy.time.Time`, which uses two 64-bit numbers for accuracy?

Reply:
The entire T&S system uses TAI as double. I see no reason to try to do better than that here.

Reply:
Typically means "always" if t0 (start_time) is a current TAI date. I'll remove "typically".